aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -169,6 +171,58 @@ Tensor& orgqr_kernel_impl(Tensor& result, const Tensor& tau, Tensor& infos, int6
   return result;
 }
 
+/*
+Solves the matrix equation op(A) X = B
+X and B are n-by-nrhs matrices, A is a unit, or non-unit, upper or lower triangular matrix
```

Comment:
If things are either "unit" or "non-unit" then is saying something is one or the other informative?

Reply:
It's saying that all diagonal entries of the matrix are assumed to be one. If `unitriangular=true` then all actual diagonal entries of the matrix are neglected and treated as one in the computations.
https://en.wikipedia.org/wiki/Triangular_matrix#Unitriangular_matrix

Reply:
That makes sense. This comment could probably be tweaked to better reflect the effect of the unitriangular argument.